README.md

```diff
@@ -189,7 +187,7 @@ The public key of the remote peer.
 
 #### `socket.publicKey`
 
-The connections public key.
+The public key of the server.
```

Comment:
This is infact the pub key of the connection itself

Reply:
Is that just english semantics? Otherwise check this:

```js
server.on('connection', function (socket) {
  // All same:
  console.log(socket.publicKey.toString('hex'))
  console.log(keyPair.publicKey.toString('hex'))
  console.log(server.publicKey.toString('hex'))

  // Both true:
  console.log(socket.publicKey === keyPair.publicKey)
  console.log(server.publicKey === socket.publicKey)
})
```